### PR TITLE
Enabled text editing on QR template #114

### PR DESCRIPTION
**Pull request by lurhays**
- Added collapsible floating panel for user to edit title, description and footer. Collapsed at page load.
- Editable pieces includes font size, font color (using picker or hex code manual entry), font type and the text itself.
- Rename bgpatterns files to aligned with its template settings.

### CODE REVIEW

**Comment by rohaizanr** [comment]
Perrrghhhh

File: dashboard/src/views/cadangan/qr/Qr.js [hunk 1]
```diff
@@ -8,11 +8,17 @@ import {
...
+  CNavItem,
+  CNavLink,
+  CTabContent,
+  CTabPane,
 } from '@coreui/react';
 import QRCode from 'react-qr-code';
 import html2canvas from 'html2canvas';
 import CIcon from '@coreui/icons-react';
 import { cilCloudDownload } from '@coreui/icons';
+import { HexColorPicker } from 'react-colorful';
```
[the remark this answers is not in the document]
Comment (lurhays) [reply dashboard/src/views/cadangan/qr/Qr.js:22]: done changes below:
- changed Pengaki to Nota Kaki.
- footer text set to "AHLI JAWATANKUASA MASJID DEMO".
- standardize all background files to PNG extension.

Pasal handling untuk mobile view, perlu masa untuk figure out.

**Comment by lurhays** [comment]
eh silap reply kat single post.

**Review by Dev4w4n — APPROVED** [review]
terima kasih

**Review by Dev4w4n — APPROVED** [review]

**Review by rohaizanr — APPROVED** [review]

**Pull request merged**